src/nwm_routing/src/nwm_routing/preprocess.py

```diff
@@ -123,8 +195,8 @@ def nwm_network_preprocess(
     if break_network_at_waterbodies:
         network_break_segments = network_break_segments.union(wbody_conn.values())
     if break_network_at_gages:
-        network_break_segments = network_break_segments.union(gages.keys())
-
+        network_break_segments = network_break_segments.union(gages['gages'].keys())
+        
```

Comment:
This is a bug fix. The `gages` dictionary object has a first-level key of `gages`. Below this, the keys are linkIDs and the values are gageIDs. Previously, we were just adding a string, `'gages'` to the set `network_break_segments`, and as a result gage segments were being left out.